app/jobs/socure_docv_results_job.rb

```diff
@@ -55,49 +55,40 @@ def perform(document_capture_session_uuid:, async: true, docv_transaction_token_
   private
 
   def record_attempt(docv_result_response:, doc_pii_response: nil)
-    image_errors = {}
-
-    if socure_doc_escrow_enabled?
-      result = fetch_images(docv_result_response.to_h[:reference_id])
-      if result.is_a?(Idv::IdvImages)
-        images = result.attempts_file_data
-      else
-        image_errors = { image_request: [:network_error] }
-      end
+    job_data = {
+      reference_id: docv_result_response.to_h[:reference_id],
+      document_capture_session_uuid:,
+      pii_from_doc: docv_result_response.pii_from_doc.to_h || {},
```

Comment:
@mitchellhenke pointed out that this would entail storing plaintext pii, which is not acceptable. will rethink this.